src/20231208_AaveV2Polygon_ReserveFactorUpdates/ReserveFactorUpdates.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Reserve Factor Updates"
+author: "TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937"
+---
+
+## Simple Summary
+
+This AIP is a continuation of [AIP 284](https://app.aave.com/governance/proposal/284) and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
+
+## Motivation
+
+This AIP will reduce deposit yield for assets on Polygon v2 by increasing the RF. With this upgrade being passed, users will be further encouraged to migrate from Polygon v2 to v3.
+
+Increasing the RF routes a larger portion of the interest paid by users to Aave DAO's Treasury. User's funds are not at risk of liquidation and the borrowing rate remains unchanged.
+
+Of the assets with an RF set at 99.99%, there is no change. All other asset reserves will have the RF increased by 5%.
+
+## Specification
+
+The following parameters are to be updated as follows:
+
+| Asset | Reserve Factor |
+| ----- | -------------- |
+| DAI   | 56.00%         |
+| USDC  | 58.00%         |
+| USDT  | 57.00%         |
+| wBTC  | 90.00%         |
+| wETH  | 80.00%         |
+| MATIC | 76.00%         |
+
+## References
+
+- Implementation: [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231208_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20231208.sol)
+- Tests: [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20231208_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20231208.t.sol)
+- [Snapshot](No snapshot for Direct-to-AIP)
+- [Discussion](https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937)
```

Comment:
while this is direct to aip, still it would be good to at least put a update post on the forum - the proposed changes are quite different to what was last written here: https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/12

Reply:
done! https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/13

I actually caught a mistake in the PR as i had MATIC going to 66% instead of 76% by writing this, thank you!